roles/kubernetes-apps/metallb/templates/metallb.yml.j2

```diff
@@ -1,3 +1,9 @@
+apiVersion: v1
```

Comment:
I've added the namespace creation at the top of this manifest.. could not see it being created in any other task

Reply:
As a mega nit could you add the document delimiter `---` at the beginning of the file?